src/business.ts

```diff
@@ -0,0 +1,121 @@
+import type { Marqeta, MarqetaOptions, MarqetaError } from './'
+
+export interface BusinessIdentifications {
+  type: string;
+  value: string;
+}
+
+export interface IncorporationType {
+  incorporationType: string;
+  stateOfIncorporation: string;
+}
+
+export interface Address {
+  address1: string;
+  address2: string;
+  city: string;
+  state: string;
+  postalCode: string;
+  country: string;
+}
+
+export interface Personal {
+  firstName: string;
+  lastName: string;
+  home: Address;
+}
+
+export interface Meta {
+  name: string;
+}
```

Comment:
I don't this is what you want to have here... In fact, you probably don't want this interface at all...

Reply:
True, changed to a [key:string]:string mapping.

Reply:
Why have:

```typescript
 export interface Meta {
   [index: string] : any
 }
```

at all? Why not just have:

```typescript
  metadata?: any;
```

in the definition of the `Business` interface?

Reply:
`<any>` will work